.github/workflows/build_all.yml

```diff
@@ -746,4 +746,4 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           repository: magma/magma
           event-type: build-all-artifact
-          client-payload: '{ "artifact": "${{ needs.agw-build.outputs.magma_package }}", "trigger_sha": "${{ github.sha }}" }'
+          client-payload: '{ "artifact": "${{ needs.agw-build.outputs.magma_package }}", "trigger_sha": "${{ github.sha }}", "commit_message": ${{ github.event.head_commit.message }} }'
```

Comment:
Out of interest, is there a reason you didn't keep the double pipe from before? I.e. why you didn't go for something like `"commit_message": ${{ github.event.head_commit.message || github.event.pull_request.title }}`

Reply:
The steps which trigger other CI jobs are only executed upon a push on master and the PR title would in my understanding only be available if the jobs would be triggered by a PR (opening, edited, closed, ...).